ui/src/main/js/page/user/user/UserCreateModal.js

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRoles } from 'store/actions/roles';
+import { saveUser, validateUser, fetchUsers } from 'store/actions/users';
+import Modal from 'common/component/modal/Modal';
+import DynamicSelectInput from 'common/component/input/DynamicSelectInput';
+import PasswordInput from 'common/component/input/PasswordInput';
+import TextInput from 'common/component/input/TextInput';
+import * as HTTPErrorUtils from 'common/util/httpErrorUtilities';
+import UserEnum from 'common/util/enums/User';
+
+const UserCreateModal = ({ data, isOpen, toggleModal }) => {
+    const dispatch = useDispatch();
+    const { external } = data;
+    const [newUser, setNewUser] = useState({});
+    
+    const fieldErrors = useSelector(state => state.users.error.fieldErrors);
+    const inProgress = useSelector(state => state.users.inProgress);
+    const roles = useSelector(state => state.roles.data);
+    const saveStatus = useSelector(state => state.users.saveStatus);
```

Comment:
```suggestion
    const fieldErrors = useSelector((state) => state.users.error.fieldErrors);
    const inProgress = useSelector((state) => state.users.inProgress);
    const roles = useSelector((state) => state.roles.data);
    const saveStatus = useSelector((state) => state.users.saveStatus);
```
Parenthesis around lambda variables is expected.